FrontEnd/FrontEnd.Web/Components/Pages/Recipes/SingleRecipe.razor

```diff
@@ -1,8 +1,11 @@
 ﻿@page "/recipes/{Id:int}"
 
+@using FrontEnd.Web.Components.CustomIcons
 @using FrontEnd.Web.Services
 @attribute [StreamRendering]
+@rendermode InteractiveServer
```

Comment:
Are you sure interactivity is needed here? I'll be able to look into this more deeply a bit later, but was wondering what did not work without it?

Reply:
It's probably because of `DeleteRecipe`, but we should be able to achieve a similar result without interactivity. I would prefer if we managed to do only with static server side rendering (this is what's used when you don't specify render mode).

Reply:
Button doesn't work if this is not specified

Reply:
Ah we're probably not losing anything with this. The initial request to load `/recipes/{id}` (should) returns all of the initial data in the HTTP request. You can disregard this